docs/tutorial/chat2.md

```diff
@@ -28,29 +31,47 @@ You should be prompted to provide a nickname for the chat session.
 Choose a nickname >>
 ```
 
-After entering a nickname, the app will randomly select and connect to a peer from the test fleet.
+After entering a nickname, the app will randomly select and connect to a peer from the `prod` fleet.
 
 ```
-No static peers configured. Choosing one at random from test fleet...
+No static peers configured. Choosing one at random from prod fleet...
+```
+
+It will then attempt to download historical messages from a random peer in the `prod` fleet.
+
+```
+Store enabled, but no store nodes configured. Choosing one at random from prod fleet...
 ```
 
 Wait for the chat prompt (`>>`) and chat away!
 
+To gracefully exit the `chat2` application, use the `/exit` [in-chat option](#in-chat-options)
+
+```
+>> /exit
+quitting...
+```
+
 ## Retrieving historical messages
 
-The `chat2` application can retrieve historical chat messages from a node supporting and running the [Waku v2 store protocol](https://specs.vac.dev/specs/waku/v2/waku-store). Just specify the selected node's `multiaddr` as `storenode` when starting the app:
+The `chat2` application can retrieve historical chat messages from a node supporting and running the [Waku v2 store protocol](https://specs.vac.dev/specs/waku/v2/waku-store), and will attempt to do so by default.
+It's possible to query a *specific* store node by configuring its `multiaddr` as `storenode` when starting the app:
 
 ```
 ./build/chat2 --storenode:/ip4/134.209.139.210/tcp/30303/p2p/16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ
 ```
 
-Alternatively, the `chat2` application will select a random `storenode` for you from the test fleet if `storenode` left unspecified.
+Alternatively, the `chat2` application will select a random `storenode` for you from the configured fleet (`prod` by default) if `storenode` is left unspecified.
 
 ```
 ./build/chat2
 ```
 
-> *NOTE: Currently (Mar 3, 2021) the only node in the test fleet that provides reliable store functionality is `/ip4/134.209.139.210/tcp/30303/p2p/16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ`. We're working on fixing this.*
+To disable historical message retrieval, use the `--store:false` option:
```

Comment:
Afaik, `store` flag has a different semantic in wakunode2, and it may get a bit confusing for those familiar with  `wakunode2`, I'd suggest to use a different name for it e.g., `fetch-history`

Reply:
`store` _should_ have the same semantic here as for `wakunode2` - in other words, if enabled it mounts `STORE` protocol. `chat2` does have the extra feature that if `STORE` is mounted and a `storenode`/`fleet` is specified it will attempt to fetch history from those nodes. In other words, if no historical messages are necessary this can be disabled by not mounting `store`.